src/framework/ui/tab/tab.component.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import {
+  View,
+  Image,
+  Text,
+  ViewProps,
+  ImageProps,
+  TextProps,
+  ImageSourcePropType,
+} from 'react-native';
+import {
+  StyledComponentProps,
+  StyleType,
+} from '@kitten/theme';
+
+interface TabProps {
+  title?: string;
+  icon?: ImageSourcePropType;
+}
+
+export type Props = TabProps & StyledComponentProps & ViewProps;
+
+export class Tab extends React.Component<Props> {
+
+  private getComponentStyle = (source: StyleType): StyleType => {
+    return {
+      container: {
+        flex: 1,
+        justifyContent: 'center',
+        alignItems: 'center',
```

Comment:
Is it in order not to create a StyleSheet object?

Reply:
We're going to inline all styles exactly in `getComponentStyle` which will be merged with consumed styles

Reply:
Got it